ms2/src/org/labkey/ms2/protein/Protein.java

```diff
@@ -44,16 +45,19 @@
 import java.util.Set;
 import java.util.TreeMap;
 
-public class Protein
+public class Protein extends SimpleProtein
 {
     private static final Logger _log = LogManager.getLogger(Protein.class);
 
-    private double _mass;
-    private String _sequence;
-    private int _seqId;
-    private String _description;
-    private String _bestName;
-    private String _bestGeneName;
+    public Protein(SimpleProtein simpleProtein)
+    {
+        setSeqId(simpleProtein.getSeqId());
```

Comment:
Suggest moving this into a new SimpleProtein constructor and making this a simple pass-through.

Reply:
Done